docs/_sass/_main.scss

```diff
@@ -116,22 +116,22 @@ h6 {
     font-size: 1.5em;
 }
 
-details summary { 
+details summary {
     cursor: pointer;
     user-select: none;
 }
 
 details > summary {
-    list-style-image: url("/assets/images/arrow-right.svg");
+    list-style-image: url('/assets/images/arrow-right.svg');
 }
 
 details[open] > summary {
-    list-style-image: url("/assets/images/down.svg");
+    list-style-image: url('/assets/images/down.svg');
 }
 
-h1, 
+h1,
 summary {
-    font-family: "ExpensifyNewKansas", "Helvetica Neue", "Helvetica", Arial, sans-serif;
+    font-family: 'ExpensifyNewKansas', 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
     font-weight: 500;
     font-size: larger;
```

Comment:
Weird that we aren't using a normal font size here. I know this isn't part of your PR but maybe we can follow up with some adjustments.

Reply:
yeah sure, created an issue - https://github.com/Expensify/App/issues/38986